### PR TITLE
Digitalpack mma frontend

### PR DESCRIPTION
**Pull request by tomverran**
Adds the JS and backend code to display the digital pack manage my account page

### CODE REVIEW

File: static/src/javascripts/projects/membership/digitalpack-tab.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        stripeForm = new PaymentForm(PAYMENT_FORM, CARD_CHANGE_SUCCESS_MSG, '/me/digitalpack-update-card');
+
+    function fetchUserDetails() {
+        ajax({
+
+            url: config.page.userAttributesApiUrl + '/me/mma-digitalpack',
+            crossOrigin: true,
+            withCredentials: true,
+            method: 'get'
+        }).then(function (resp) {
```
Comment (afiore) [inline static/src/javascripts/projects/membership/digitalpack-tab.js]: Shouldn't we handle errors here?

Comment (tomverran) [inline static/src/javascripts/projects/membership/digitalpack-tab.js]: Well this behaviour is consistent with the membership version of this page
I do agree we should handle errors though

File: static/src/javascripts/projects/membership/digitalpack-tab.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            $(REMAINING_TRIAL_LENGTH_CONTAINER).removeClass(IS_HIDDEN_CLASSNAME);
+        }
+
+        $(PACKAGE_NEXT_PAYMENT_DATE).text(formatters.formatDate(userDetails.subscription.nextPaymentDate));
+        if (userDetails.subscription.nextPaymentPrice != userDetails.subscription.plan.amount) {
+            $(PACKAGE_NEXT_PAYMENT_PRICE).text(formatters.formatAmount(userDetails.subscription.nextPaymentPrice, glyph));
+            $(PACKAGE_NEXT_PAYMENT_CONTAINER).removeClass(IS_HIDDEN_CLASSNAME);
+        }
+
+        if (!userDetails.optIn) {
```
Comment (afiore) [inline static/src/javascripts/projects/membership/digitalpack-tab.js]: What does 'optIn' actually mean?

**Comment by afiore** [comment]
:+1: perhaps error handling is for another PR

**Comment by afiore** [comment]
:+1: again...

**Comment by mario-galic** [comment]
:+1: [Acceptance criteria](https://trello.com/c/EbX1W3hI/214-manage-my-account-update-credit-card) satisfied in DEV env.

**Pull request merged**